CorsixTH/Lua/diseases/serious_radiation.lua

```diff
@@ -43,6 +43,7 @@ disease.initPatient = function(patient)
   patient:setLayer(1, math.random(0, 3) * 2)
   patient:setLayer(3, 0)
   patient:setLayer(4, 0)
+  patient.th:setOverlay(AnimationOverlayFlags.Glowing)
```

Comment:
Can't add my comment at line 1. I know most of your work is in other files but as a summation of that I would recommend changing the top to:
``--[[ Copyright (c) 2010 Manuel "Roujin" Wolf, 2020 <name> "driverpt" <surname>``
(Fill name and surname of course only if you want to)

Reply:
Certainly in LICENSE.txt - if you don't, I will before the next release. Here I don't think it's justified, but files with more substantial changes, certainly.

Reply:
I think VCS will do that for you

Reply:
@TheCycoONE , i'm already there

```
Copyright (C) 2012-2013 Luis "Driver" Duarte
```

Reply:
Ah, welcome back, just need to add a `, 2020` or change `2013` to `2020` - up to you.